learn/hybrid-mobile/export-cordova-zip.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Export Cordova zip"
+id: ""
+---
+---
+
+A Cordova zip will be used to create a mobile application. WaveMaker allows you to export a cordova zip from a *mobile project* so that
+you can use it to generate a *mobile application*.
+
+Following are the two approaches which can be followed to create a cordova zip 
+
+## 1. Using WaveMaker studio
+
+1. In the top right menu bar of WaveMaker Studio, export menu is present.  Click on 'Download Cordova zip' menu item.
+[![](/learn/assets/Cordova_Zip.png)](/learn/assets/Cordova_Zip.png)
+
+2. A dialog with all the cordova configuration will open.
+[![](/learn/assets/Build_Cordova_Zip.png)](/learn/assets/Build_Cordova_Zip.png) 
+
+3. Provide all necessary configuration and click on 'build' button. A job gets posted to prepare a cordova zip.
```

Comment:
We can talk about few important configuration options to review (like the server url or so).
Also point to another document where the configuration options are talked about.